edxval/utils.py

```diff
@@ -3,13 +3,16 @@
 """
 
 from __future__ import absolute_import
+
 import json
+
 from django.conf import settings
 from django.core.files.storage import get_storage_class
 from fs.path import combine
 from pysrt import SubRipFile
 
 
+# pylint: disable=missing-docstring
```

Comment:
Again, I'd rather avoid module-wide warning disables like this if we can.  The latest `pylintrc` from edx-lint should be a little less strict about which methods need docstrings.